tests/testthat/test-find_columns.R

```diff
@@ -34,11 +34,6 @@ test_that("find_columns works as expected", {
     c("gear", "cyl")
   )
 
-  expect_equal(
-    find_columns(iris, Spec),
-    "Species"
-  )
-
```

Comment:
I don't think it's a good idea to support something like this, which is why I deleted it. It is safer to use "Spec" and `regex = TRUE`